Mike

MikeMosc opened the pull request.

MikeMosc committed:
Added the FireRules Class (Not fully funct.) #39

MikeMosc committed:
Added the FireRules Class (Not fully funct.) #39

MikeMosc committed:
Can associate data w/ one drl file at a time

MikeMosc committed:
Can associate data w/ one drl file at a time

01SVTCobra committed:
Complete overhaul of file parser. Included support for JSON files while also maintaining and streamlining
support for generic text files. Included GSON .jar file.
- Users are now able to import basic JSON files and have its data stored.
@author - Trae X. Lewis

MikeMosc committed:
Slight modification to classes

MikeMosc committed:
Slight modification to classes

01SVTCobra reviewed (approved):
Make sure you go through and thoroughly document the code. Have no clue whats going on in FileRules class. Also naming convention for "FireRules" could be like "RuleActivator" or RuleActivation. Food for thought.

01SVTCobra commented:
Because there are conflicts I will pull your branch and merge it into master manually.

01SVTCobra committed:
Added support for reading JSON rule files in InputReaderService.
Fixed formatting issues in Action and ActionCollectionServices.
Added Code_Resources.zip which includes the GSON.jar

01SVTCobra committed:
Added support for reading JSON rule files in InputReaderService.
Fixed formatting issues in Action and ActionCollectionServices.
Added Code_Resources.zip which includes the GSON.jar

01SVTCobra committed:
Removed ~$tWorkflow.docx

01SVTCobra committed:
Removed extra Condition.java class in helpers package

01SVTCobra committed:
Removed unnecessary .java class files

01SVTCobra committed:
Removed unnecessary .java class files

01SVTCobra committed:
Merge remote-tracking branch 'refs/remotes/origin/txl_branch'

Conflicts:
	Rules Based Decision Engine/src/controllers/Driver.java
	Rules Based Decision Engine/src/helpers/Condition.java
	Rules Based Decision Engine/src/services/InputReaderService.java

01SVTCobra committed:
Merged branch txl_branch into txl_branch

01SVTCobra committed:
Merged txl_branch into master

01SVTCobra committed:
Merge remote-tracking branch 'refs/remotes/origin/Mike'

Conflicts:
	Rules Based Decision Engine/src/controllers/Driver.java

01SVTCobra committed:
Revert "Merge remote-tracking branch 'refs/remotes/origin/Mike'"

This reverts commit 6c1056ddb37838f6b59fe6468a7694cb67e727dc.
